server/src/main/java/io/spine/server/procman/ProcessManagerRepository.java

```diff
@@ -263,12 +261,8 @@ void dispatchNowTo(I id, EventEnvelope event) {
     }
 
     @Override
-    public void onError(CommandEnvelope envelope, RuntimeException exception) {
-        CaughtError error = commandErrorHandler.handleError(envelope, exception);
-        error.asRejection()
-             .map(RejectionEnvelope::getOuterObject)
-             .ifPresent(event -> postEvents(ImmutableList.of(event)));
-        error.rethrowOnce();
+    public void onError(CommandEnvelope cmd, RuntimeException exception) {
```

Comment:
Please document this method. It used to inherit documentation (and behaviour) from a parent type. Now it's on its own.

Please apply such a change to other similar cases as well.